server/src/server.ts

```diff
@@ -1,21 +1,170 @@
 import express, { Application } from 'express';
 import cors from 'cors';
+import cookieSession from 'cookie-session';
 import dotenv from 'dotenv';
-import recordRouter from './routes/record';
-import { connectToServer } from './db/connection';
-
+import route from './routes/registerRoute'
+import routes from './routes/loginRoute';
+import routee from './routes/currentUser'
+import { connect } from './db/connection';
+import { createUser, getEmail, getUsers, User, workoutModel } from './model';
+import bcrypt from 'bcrypt';
 dotenv.config({ path: './config.env' });
+import jwt from 'jsonwebtoken'
+const secretKey: string = process.env.SECRET_KEY || '';
+
+
 
 const app: Application = express();
 const port: string | number = process.env.PORT || 7036;
 
 app.use(cors());
+app.use(routes);
+app.use(route);
+app.use(routee)
+app.set('trust proxy', true) // trust first proxy
 app.use(express.json());
-app.use(recordRouter);
+
+app.use(cookieSession({
+    signed: false,
+    name:'session',
+    maxAge: 30 * 24 * 60 * 60,
+    keys: ['secretKey']
+})
+);
+
 
 app.listen(port, () => {
-    connectToServer((err: Error) => {
-        if (err) console.error(err);
-    });
+    connect();
     console.log(`Server is running on port: ${port}`);
 });
+
+
+
+export const register = async(req: express.Request, res: express.Response) =>{
+    connect()
+    try {
+       
+        // const { email, password, name } = req.body
+        const { email, password, name } = {
```

Comment:
I am not sure if this is example code, but make sure no example code etc. is in final pull request - can cause issues